Makefile

```
@@ -19,13 +20,17 @@ $(WEB_SLANG_SOURCE):
 $(WEB_SLANG_FILE): $(WEB_SLANG_SOURCE)
 	./zhwiki-web-slang.py --process $(WEB_SLANG_SOURCE) > $(WEB_SLANG_FILE)
 
+$(EXCLUDE_TITLES_FILE):
+	./exclude_titles.py || true
```

```suggestion
	./exclude_titles.py
```
Silent error is not good.